src/components/signup/PhoneForm.web.js

```diff
@@ -119,7 +119,7 @@ class PhoneForm extends React.Component<Props, State> {
                 {`${getFirstWord(fullName)},\nenter your phone number\nso we can verify its you`}
               </Section.Title>
             </Section.Row>
-            <Section.Stack justifyContent="center" style={styles.column}>
+            <Section.Stack className="signup_phone_input" justifyContent="center" style={styles.column}>
```

Comment:
1. this requires you to have two almost identical components for web and native. Actually it's not the case enough to diversify PhoneForm - as we're not faced with native (or web) specific case (like in FR where on web we should import & use Zoom web SDK but on mobile we should access exported functions from NativeModules and prefill some params e.g. jwt, good server url)
2. this fixes the issue partially (in this form only). The correct solution will be to fix it on the `<PhoneInput>` component level. It also allows you to skip using CSS and you could have simple implementation

Reply:
@serdiukov-o-nordwhale 
1) right, this implementation will work only for the web. I checked how the phone number input implemented in the react-native branch for native apps - there is already a separate file created which implements phone input for native apps, as it works differently as on web. ss I think it is ok.

Reply:
Ok